universal/update.md

```diff
@@ -51,12 +51,11 @@ So the process goes as follows:
 
 ![](../images/post-install/update-md/oc-config-compare.png)
 
-* Once you've made the adjustments, to make sure that you config is compliant with the newest release of OpenCore, use OpenCore Utility ocvalidate: it'll tell you whether there are errors in your config.
-    * Please note, that `ocvalidate` must match the used OpenCore release and may not be able to detect all configuration flaws present in the file.
-If you want to get a bit more technical read the [Differences.pdf](https://github.com/acidanthera/OpenCorePkg/blob/master/Docs/Differences/Differences.pdf).
+* Once you've made the adjustments, to make sure that you config is compliant with the newest release of OpenCore, you can the OpenCore Utility ocvalidate: this tool will help ensure your config.plist is matching the OpenCore specification of the matching build.
```

Comment:
@khronokernel I think that you missed a verb:

* Once you've made the adjustments, to make sure that you config is compliant with the newest release of OpenCore, you can **use** the OpenCore Utility ocvalidate: this tool will help ensure your config.plist is matching the OpenCore specification of the matching build.

Reply:
Ah thank you, will push that fix to master in a sec